packit_service/worker/jobs.py

```diff
@@ -615,7 +615,8 @@ def get_jobs_matching_event(self) -> List[JobConfig]:
                 )
                 and job not in jobs_matching_trigger
             ):
-                jobs_matching_trigger.append(job)
+                if not job.manual:
```

Comment:
I assume there needs to be some additional condition, something like:
```python
if not job.manual or self.event in MANUAL_EVENTS:
```

(where the MANUAL_EVENTS would be list of events for comments/check re-runs)

But once we have a testcase, that will be clearer